scripts/local-testing.sh

```diff
@@ -0,0 +1,22 @@
+#! /bin/bash
+
+get_latest_release() {
+    curl --silent "https://api.github.com/repos/$1/releases/latest" | # Get latest release from GitHub api
+    grep '"tag_name":' |                                            # Get tag line
+    sed -E 's/.*"([^"]+)".*/\1/' |                                  # Pluck JSON value
+    sed 's/v//'                                                     # Remove "v" prefix
+}
+
+LATEST_TAG=$(get_latest_release snyk/kubernetes-monitor)
+IP=$(ifconfig | grep "inet " | grep -Fv 127.0.0.1 | awk 'NR==1{print $2}')
+
+echo "Enter your integration ID: "
+read INTEGRATION_ID
```

Comment:
Can we just read from environment?

Reply:
I think so, when missing?

Reply:
I despise anything that prompts me in the terminal. I think `read` is find if there is no `INTEGRATION_ID` in the env though.